chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -141,7 +141,7 @@ spec:
           {{- include "custom_airflow_environment" . | indent 10 }}
           {{- include "standard_airflow_environment" . | indent 10 }}
         {{- end }}
-        {{- if .Values.dags.gitSync.enabled }}
+        {{- if and (not .Values.dagProcessor.enabled) .Values.dags.gitSync.enabled }}
```

Comment:
Don't we still need this if we have dagProcessor and local?

Reply:
Makes sense, local executor still requires DAGs to be mounted.